### DISCUSSION TITLE
ModuleNotFoundError: No module named 'tacotron_cleaner'

### DISCUSSION THREAD

**Discussion by wwyl2000** [espnet/espnet#4506] [Q&A]
Hi

I am trying to use ~/work/espnet/egs2/librispeech/asr1
and i got error:
ModuleNotFoundError: No module named 'tacotron_cleaner'
I cannot find module tacotron_cleaner. Did I missed any parts, or any problem with settings somewhere? 

In espnet, i found:
./espnet/nets/pytorch_backend/tacotron2
./espnet2/tts/tacotron2
./espnet2/tts/tacotron2/tacotron2.py

Thanks for your help in advance.

wwy

**Comment by sw005320** [comment]
Did you set the path correctly via path.sh?

**Reply by wwyl2000** [reply]
Could you please tell me how to set that?
At ~/work/espnet/egs2/librispeech/asr1, i can see:
path.sh -> ../../TEMPLATE/asr1/path.sh*

Thanks.

**Reply by sw005320** [reply]
If you execute it through `run.sh`, you can import paths from `path.sh` (via https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/asr.sh#L263)

**Reply by wwyl2000** [reply]
I am using ~/work/espnet/egs2/librispeech/asr1/run.sh which calls asr.sh (in the same directory), and i think it is the same as the asr.sh you sent above in the link. 
In  my asr.sh, on line 263, it is 
. ./path.sh

So i assume i have imported paths already?
Which part in ESPnet performs the installation of tacotron_cleaner?
Thank again!

**Reply by kan-bayashi** [reply]
https://github.com/espnet/espnet/blob/f2778f798b76e102602078213a21b63aa592be70/setup.py#L35
It should be installed here.

**Reply by wwyl2000** [reply]
Thanks Shinji and Tomoki. I pip installed the package and the problem was resolved.
